js/constants/constants.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
     ERROR_SCREEN: "errorScreen"
   },
 
+  CUSTOM_EVENTS: {
```

Comment:
But yeah, I'm also using custom events so that they're private. This new event is internal to the skin so the customer shouldn't be using it.